docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Pod Memory and CPU Resources"
+date: 2020-06-30T08:55:00-05:00
+draft: false
+weight: 25
+---
+
+### Contents
+
+ - [Introduction](#introduction)
+ - [Setting resource requests and limits in a domain resource](#setting-resource-requests-and-limits-in-a-domain-resource)
+ - [Determining pod Quality Of Service](#determining-pod-quality-of-service)
+ - [Java heap size and memory resource considerations](#java-heap-size-and-memory-resource-considerations)
+   - [Overview](#overview)
+   - [Default heap sizes](#default-heap-sizes)
+   - [Configuring heap size](#configuring-heap-size)
+ - [CPU resource considerations](#cpu-resource-considerations)
+ - [Operator sample heap and resource configuration](#operator-sample-heap-and-resource-configuration)
+ - [Configuring CPU affinity](#configuring-cpu-affinity)
+ - [Measuring JVM heap, pod CPU, and pod memory](#measuring-jvm-heap-pod-cpu-and-pod-memory)
+ - [References](#references)
+
+### Introduction
+
+An operator creates a pod for each WebLogic Server instance and each pod will have a container. It's important that the container has enough resources in order for WebLogic to run efficiently.
+
+If a pod is scheduled on a node with limited resources, then it's possible for node to run out of memory or CPU resources, and for the pod's applications to stop working properly or have degraded performance. It's also possible for a rogue application to use all of a node's available memory and/or CPU, which makes other containers running on the same node unresponsive. The same problem can happen if an application has a memory leak.
+
+You can solve these problems by configuring resource requests and limits for your pods. A resource limit prevents a pod from using more than its share of a resource. Thus, it improves reliability, stability, and helps with hardware capacity planning. Additionally, resource requests and limits determine a pod's Quality of Service.
+
+### Setting resource requests and limits in a domain resource
+
+You can set Kubernetes memory and CPU requests and limits in a [domain resource]({{< relref "/userguide/managing-domains/domain-resource" >}}) using its `spec.serverPod.resources` stanza, and you can override the setting for individual WebLogic servers or clusters using the `serverPod.resources` element in `spec.adminServer`, `spec.clusters`, and/or `spec.managedServers`. For example: 
```

Comment:
This is good. Let's also link to the standard Kubernetes doc. topics https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/ and https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/.

Reply:
Included above additional links for Kubernetes doc.